spec/sections/copies.bs

```diff
@@ -251,7 +251,35 @@ dictionary GPUImageCopyExternalImage {
     : <dfn>source</dfn>
     ::
         The source of the [=image copy=]. The copy source data is captured at the moment that
-        {{GPUQueue/copyExternalImageToTexture()}} is issued.
+        {{GPUQueue/copyExternalImageToTexture()}} is issued. Source size is defined by source
+        type, given by this table:
+        
+        <table class=data>
+            <thead>
+                <tr>
+                    <th>Source type
+                    <th>Width
+                    <th>Height
+            </thead>
+            <tbody>
+                <tr>
+                    <td>{{ImageBitmap}}
+                    <td>{{ImageBitmap/width|ImageBitmap.width}}
+                    <td>{{ImageBitmap/height|ImageBitmap.height}}
+                <tr>
+                    <td>{{HTMLVideoElement}}
+                    <td>{{HTMLVideoElement/videoWidth|HTMLVideoElement.videoWidth}}
+                    <td>{{HTMLVideoElement/videoHeight|HTMLVideoElement.videoHeight}}
```

Comment:
I previously suggested copying language from the WebGL spec, but found that it doesn't seem to have any. It just occurred to me we can copy the language used by the 2d canvas (HTML) spec. I'd like to do that because I have a vague recollection that these attributes themselves might not stay perfectly up to date with the bitmap we get from the video, due to the asynchronous nature of videos.

https://html.spec.whatwg.org/multipage/canvas.html#image-sources-for-2d-rendering-contexts
> When a CanvasImageSource object represents an HTMLVideoElement, then the frame at the current playback position when the method with the argument is invoked must be used as the source image when rendering the image for CanvasRenderingContext2D APIs, and the source image's dimensions must be the intrinsic width and intrinsic height of the media resource (i.e., after any aspect-ratio correction has been applied).

So here I think we can just copy this terminology of "intrinsic width/height".

Reply:
Thanks for the comments, address this.